pkg/minikube/machine/cache_images.go

```diff
@@ -259,20 +264,26 @@ func DoLoadImages(images []string, profiles []*config.Profile, cacheDir string)
 func transferAndLoadCachedImage(cr command.Runner, k8s config.KubernetesConfig, imgName string, cacheDir string) error {
 	src := filepath.Join(cacheDir, imgName)
 	src = localpath.SanitizeCacheDir(src)
-	return transferAndLoadImage(cr, k8s, src)
+	return transferAndLoadImage(cr, k8s, src, imgName)
 }
 
 // transferAndLoadImage transfers and loads a single image
-func transferAndLoadImage(cr command.Runner, k8s config.KubernetesConfig, src string) error {
+func transferAndLoadImage(cr command.Runner, k8s config.KubernetesConfig, src string, imgName string) error {
 	r, err := cruntime.New(cruntime.Config{Type: k8s.ContainerRuntime, Runner: cr})
 	if err != nil {
 		return errors.Wrap(err, "runtime")
 	}
+
+	if err := r.RemoveImage(imgName); err != nil {
+		klog.Warningf("remove image: %v", err)
```

Comment:
if image doesnt exist it will always warn ? in case the error is doesnt exist we could skip warn ?